demisto_sdk/__main__.py

```diff
@@ -217,9 +217,10 @@ def unify(**kwargs):
 
 
 # ====================== validate ====================== #
-@main.command(name="validate",
-              short_help='Validate your content files. If no additional flags are given, will validated only '
-                         'committed files')
+@main.command(
```

Comment:
where is the command name?

Reply:
You don't need it. it uses the command name below as the command name. (it replaces `_` with `-`)